src/main/java/mops/foren/domainmodel/Role.java

```diff
@@ -0,0 +1,10 @@
+package mops.foren.domainmodel;
+
+import java.util.Collection;
+
+public interface Role {
+
+    boolean hasPermission(Permission permission);
+
+    boolean hasPermissions(Collection<Permission> permissions);
```

Comment:
What is the purpose of this function ?
Is there a case in which a class has to give out all its permissions and not just answer if it has a certain permission ?